src/data/javascript-quiz.ts

```diff
@@ -1583,6 +1583,17 @@ const javascriptQuiz = [
       "Prototypes provide the means for JavaScript objects to inherit features from other objects.",
     Link: "https://www.freecodecamp.org/news/javascript-prototype-explained-with-examples/",
   },
+   {
+    Question:
+      "In JavaScript, how many threads does code run in?",
+    Answer: "Single thread",
+    Distractor1: "Multiple threads",
+    Distractor2: "No threads",
+    Distractor3: "Can be both single threaded and multi-threaded",
+    Explanation:
+      "JavaScript is a single-threaded language because while running code on a single thread, it can be really easy to implement as we don't have to deal with the complicated scenarios that arise in the multi-threaded environment like deadlock",
```

Comment:
Please don't use direct quotes from external sites like these:
https://www.geeksforgeeks.org/why-javascript-is-a-single-thread-language-that-can-be-non-blocking/#.

```suggestion
      "JavaScript is a single-threaded language because it has one call stack and memory heap.",
```